### PR TITLE
Add med libraries

### PR DESCRIPTION
**Pull request by jschueller**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jschueller** [comment]
ping @conda-forge/help-c-cpp

File: recipes/salome-medcoupling/build.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+git clone --depth 1 -b V9_11_0 https://git.salome-platform.org/gitpub/tools/configuration.git ${SRC_DIR}/configuration
+
```
Comment (xhochy) [inline recipes/salome-medcoupling/build.sh:4]: Please include this as additional sources in the recipe.

File: recipes/salome-med/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - {{ compiler('fortran') }}  # [unix]
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+  host:
+    - python
+    - hdf5
+  run:
+    - python
+
+test:
```
Comment (xhochy) [inline recipes/salome-med/meta.yaml:36]: Please add a pip check call

**Comment by Krande** [comment]
Hey, @jschueller! 

Have you seen the existing `libmed` feedstock https://github.com/conda-forge/libmed-feedstock which also represents the `med` library? Is there something that might be missing in the `libmed` feedstock? If so we would be happy to collaborate on making improvements. 

FYI: We are also making an effort to add `medcoupling` (see https://github.com/conda-forge/staged-recipes/pull/24378)

**Comment by jschueller** [comment]
see https://github.com/conda-forge/staged-recipes/pull/24378

**Comment by jschueller** [comment]
hi, for libmed it seems my patch was way simpler than yours

**Comment by Krande** [comment]
@jschueller That's interesting! If you could create a PR for the libmed feedstock with your simplified patches we can take a look more closely on the differences there?

**Pull request closed without merging**